app/src/main/java/tk/wasdennnoch/androidn_ify/XposedHook.java

```diff
@@ -49,7 +49,6 @@
     public static final String SETTINGS_OWN = PACKAGE_OWN + ".ui.SettingsActivity";
 
     public static final String ACTION_MARK_UNSTABLE = "tk.wasdennnoch.androidn_ify.action.ACTION_MARK_UNSTABLE";
-    public static final String GOOGLE_APP_VERSION_REGEX = "6\\.6\\.1[46]\\.21\\.[a-z]+[0-9]*"; // gotta love regex
```

Comment:
@wasdennnoch your comment broke my rebase, had to fix conflicts.. aka remove that line again -.- :DD